fr/core-libraries/caching.rst

```diff
@@ -38,10 +38,10 @@ votre propre système de mise en cache. Les moteurs de cache intégrés sont:
   est une extension PHP qui fournit des fonctionnalités similaires à APC.
 * ``MemcachedEngine`` Utilise l'extension
   `Memcached <http://php.net/memcached>`_. Il est aussi une interface avec
-  memcache mais il fournit une meilleur performance.
+  Memcached mais il fournit une meilleur performance.
```

Comment:
By the way, there is also a sentence in [the api](http://api.cakephp.org/3.0/class-Cake.Cache.Engine.MemcachedEngine.html) that might need a change::
    
    Main advantage of this Memcached engine over the memcached engine... ?